src/sqlfluff/core/rules/__init__.py

```diff
@@ -6,11 +6,25 @@
 
 std_rule_set = RuleSet(name="standard", config_info=STANDARD_CONFIG_INFO_DICT)
 
+# Sphinx effectively runs an import * from this module in rules.rst, so initialise
+# __all__ with an empty list before we populate it with the rule names.
+__all__ = []
+
 # Iterate through the rules list and register each rule with the std_rule_set
 for plugin_rules in get_plugin_manager().hook.get_rules():
     for rule in plugin_rules:
         std_rule_set.register(rule)
 
+        if not rule.__name__.startswith("Rule_Example"):
```

Comment:
Exclude the Example rules from the "Example" plugin

Reply:
It feels a little strange for the core code to have to do this. Is this code here mainly to fix the documentation, or does it serve some other purpose?

Is there potentially another way to do this? For example, if it's mainly for the docs, could we `pip uninstall` the example plugin before building the docs?

Reply:
yeah I agree - I'm also not sure this is required 🤔 . readthedocs checks out the repo and just installs sqlfluff directly. It doesn't use the `tox.ini` file, and that's the only place I cans see that we install `plugins/sqlfluff-plugin-example`. My guess is that even if we remove this we still wont see the example rule in the docs.